packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHomepageNavBar.tsx

```diff
@@ -87,7 +87,12 @@ export const SynapseHomepageNavBar: React.FunctionComponent<
   const handleClosePortalResources = () => {
     setPortalResourcesAnchorEl(null)
   }
-  const oneSageURL = useOneSageURL()
+
+  const onLoginClicked = () => {
+    // Navigate to the LoginPlace, which will set a cookie that will be used to redirect back from OneSage
+    gotoPlace(LOGIN_LINK)
+  }
```

Comment:
Setting this cookie is crucial to allow login in the dev environment, since the [syn45291362 table](https://www.synapse.org/Synapse:syn45291362/tables/) doesn't exist when using another backend stack.